server/routes/api.py

```diff
@@ -1,9 +1,30 @@
 from flask import Blueprint, jsonify, make_response
+from firebase_admin import auth, credentials
+import firebase_admin
+from flask import request
 
 
 bp = Blueprint('api', __name__, url_prefix='/service')
 
+cred = credentials.Certificate("./tiktaktoe-private-key.json")
+firebase_admin.initialize_app(cred)
 
 @bp.route('/hello_world')
 def hello_world():
     return make_response(jsonify(message='Hello World')), 200
+
+@bp.route('/signup')
+def test():
```

Comment:
This method shouldn't be called `test`. `signup` seems more appropriate.

Reply:
oh yeah, the route used to be test and when I updated it, I forgot to change that one too

Reply:
So I decided to call this test because the signin/signup does not happen in the backend. Firebase takes care of that

Reply:
How about `firebase-authentication`? `test` just seems really broad.